core/src/main/java/io/confluent/rest/RestConfig.java

```diff
@@ -526,6 +527,28 @@ public class RestConfig extends AbstractConfig {
       "Maximum buffer size for jetty request headers in bytes";
   protected static final int MAX_REQUEST_HEADER_SIZE_DEFAULT = 8192;
 
+  protected static final String NETWORK_TRAFFIC_RATE_LIMIT_ENABLE_CONFIG =
+      "network.traffic.rate.limit.enable";
+  protected static final String NETWORK_TRAFFIC_RATE_LIMIT_ENABLE_DOC =
+      "Whether to enable network traffic rate-limiting. Default is false";
+  protected static final boolean NETWORK_TRAFFIC_RATE_LIMIT_ENABLE_DEFAULT = false;
+
+  protected static final String NETWORK_TRAFFIC_RATE_LIMIT_BACKEND_CONFIG =
+      "network.traffic.rate.limit.backend";
+  protected static final String NETWORK_TRAFFIC_RATE_LIMIT_BACKEND_DOC =
+      "The rate-limiting backend to use. The options are 'guava', and 'resilience4j'."
+          + "Default is 'guava'.";
```

Comment:
For reference, the kafka-rest default may be guava (and good we match it), but we actually run with the config set to resliance4j, so launch darkly should match.

Reply:
Yeah, I'll make the config in LD set to resilience4j

Reply:
What is the reason to keeping both guava/resilience4j as 2 options? I never understood why kafka-rest gives an option. 
IMO, ideally whatever the reason, would have converged to 1 option by now, no reason to have LD/config.

Reply:
it is some historical reason I think when it first developed